pyproject.toml

```diff
@@ -15,6 +15,8 @@ emoji = "^2.1.0"
 emojificate = "^0.6.0"
 streamlit-ace = "^0.1.1"
 streamlit-talk = "^0.0.3"
+isort = "^5.10.1"
+autoflake = "^1.7.7"
```

Comment:
[nit] should these be dev dependencies?

Reply:
Yes! Moved to dev deps here [9f7e4de](https://github.com/cohere-ai/sandbox-conversant-lib/pull/64/commits/9f7e4de71fd2b7f7cac08bce1c720e723ff3ddd4)